### PR TITLE
Feature/core/copy core to android core

### PR DESCRIPTION
**Pull request by TalhaAli00**

### CODE REVIEW

**Review by jakobuid — APPROVED** [review]

File: android_core/src/main/kotlin/com/walletconnect/android_core/common/model/SymmetricKey.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.walletconnect.android_core.common.model
+
+import com.walletconnect.foundation.common.model.Key
+
+@JvmInline
+value class SymmetricKey(override val keyAsHex: String) : Key
```
Comment (Elyniss) [inline android_core/src/main/kotlin/com/walletconnect/android_core/common/model/SymmetricKey.kt:6]: Why is this moved from foundation?

Comment (TalhaAli00) [reply android_core/src/main/kotlin/com/walletconnect/android_core/common/model/SymmetricKey.kt:6]: SymmetricKey is not used in Foundation so I moved it up into Core
